### ISSUE TITLE
Classify and record misconceptions in answers

### ISSUE THREAD

**Issue by dbosk** [lenfra/examtool#30]
When the students answer wrongly, there might be some systematic misconception behind it.

At the moment it's possible to see how the students perform on the question. Also how students perform on a particular ILO. This is quantitative and nice, but it doesn't give any information on possible causes (*qualitative* properties).

However, it would be nice with some qualitative aspects too. For instance, it would be nice to record categories for the students' wrong answers, e.g. thought that routing happens on the data-link layer.

This would require a nice interface when grading. Particularly one that can present these categories and allow searching through them. And, of course, add a new category if needed.

Then these categories could be visualized in some nice way. There are probably a variety of alternatives here.

**Comment by lenfra** [comment]
Misconceptions could be recorded in the solution section beneath the suggested answer, but then they wouldn't be indexed in the same way, that is, be able to retrieve common misconceptions per exam (are the same misconceptions occurring year after year?)
Interesting idea!

**Comment by dbosk** [comment]
Yes, exactly. The idea is to make such analyses more easy. (One could just record them on paper too, but that would also decrease the ease of analysis.) One would want to be able to check if it's just the question, or if it's related to the ILO (and hence more systematic).
